ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,121 @@
+  <script type="text/javascript" src="js/setup.js?@(resource_hashes['setup.js'])"></script>
+  <script type="text/javascript">
+    var QUERY_TRANSFORMS = {
+      'id': 'id=".*"',
+      'label': 'label=".*"',
+    };
+
+    function encoded_query_a_tag(query, title, content) {
+      document.write('<a href="?q=' + encodeURIComponent(query) + '" title="' + title + '">' + content + "</a>")
+    }
+  </script>
+
+  <link rel="stylesheet" type="text/css" href="css/status_page.css?@(resource_hashes['status_page.css'])" />
+  <link rel="stylesheet" type="text/css" href="css/blocked_releases_page.css?@(resource_hashes['blocked_releases_page.css'])" />
+</head>
+<body>
+@{
+import time
+}@
+  <script type="text/javascript">
+    window.body_ready_with_age(moment.duration(moment() - moment("@(time.time())", "X")));
+  </script>
+  <div class="top logo">
+    <h1><img src="http://wiki.ros.org/custom/images/ros_org.png" alt="ROS.org" width="150" height="32" /></h1>
+    <h2>Repos blocked by other repos<br />@(rosdistro_name)</h2>
+  </div>
+  <div class="top search">
+    <form action="?">
+      <input type="text" name="q" id="q" />
+      <p>
+        <a href="?q=" title="Show all repos">all</a>,
+        <script language="JavaScript">encoded_query_a_tag('label="RELEASED"', "Repositories which have already been released", "released")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNRELEASED"', "Repositories which have not been released", "unreleased")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="BLOCKED"', "Repositories which are blocked from being released because of unreleased dependencies", "blocked")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED"', "Repositories which can be released", "releasable")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_BLOCKING"', "Repositories which can be released and are preventing others from being released", "releasable and blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_UNBLOCKING"', "Repositories which can be released and are not preventing others from being released", "releasable and not blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('id="metapackages"', "Repositories which are dependencies of the desktop-full metapackage", "desktop-full")</script>
+      </p>
+      <p id="search-count"></p>
+    </form>
+  </div>
+
+  <div class="top age">
+    <p></p>
+  </div>
+
+  <div class="table-div">
+    <table>
+      <thead>
+        <tr>
+          <!-- warning: if titles run onto two lines, first row of table data may be affected -->
+          <th class="sortable"><div>Repository</div></th>
+          <th class="sortable"><div>Version</div></th>
+          <th class="sortable">
+            <div title="Number of repositories which are directly blocking this one by not being released">
```

Comment:
I found the current title to be difficult to understand - especially since "by not being released" follows at the end but refers to the first part of the sentence. I would suggest the following: 

> Number of unreleased repositories which are directly blocking this one

I will add similar comments to the other columns.